cibyl/cli/main.py

```diff
@@ -79,6 +81,26 @@ def setup_output_format(args):
         raise InvalidArgument(msg) from None
 
 
+def enable_plugins(plugins=None):
+    """Enable plugins based on a given list of plugins"""
+    if plugins:
+        plugin_sources = []
+        for plugin in plugins:
+            plugin_module = extend_models(plugin)
+            path = os.path.join(os.path.dirname(os.path.abspath(
+                plugin_module.__file__)), 'sources')
+
+            for py in [f[:-3] for f in os.listdir(path)
+                       if f.endswith('.py') and f != '__init__.py']:
+                mod = __import__('.'.join(
+                    [f"cibyl.plugins.{plugin}.sources", py]), fromlist=[py])
+                plugin_sources.extend(
+                    [getattr(mod, x) for x in dir(mod)
```

Comment:
This currently includes any class imported inside the plugin sources. IMO it could be made a bit more readable and filter only the classes defined in the plugin sources using the inspect module and the `__module__` variable. See [this SO](https://stackoverflow.com/a/8093671) answer for example

Reply:
I've modified it, but not sure 100% if that's what you meant. Let me know

Reply:
Yes, in my opinion is clearer now, thank you!